README.md

```diff
@@ -70,6 +70,6 @@ Remove containers:
 
 Angular app located at [localhost:4200](http://localhost:4200)
 
-### Run pre-commit:
+### Run pre-commit
```

Comment:
We generally do Markdown headings `## Like this ##` with trailing octothorpes.  Can you make this change throughout this file?

I'm also confused as to why the linters aren't checking this file.  @mcdonnnj?

Reply:
We don't have anything auto-formatting on Markdown files. We would need to expand the `markdownlint` configuration to enforce this rule because it is on the default `consistent`. We should really look at adding a number of rules to the `markdownlint` configuration to better enforce our preferring Markdown style.

Reply:
@mcdonnnj I believe it was automatically enforcing it when I ran precommit for the python-library-skeleton, but it doesn't do it in the skeleton-docker.

Reply:
@zenine07 The default of `consistent` will only flag if you are using different styles. Since you did not merge in the skeleton's README you are consistently using the ATX style (and not the ATX closed style we prefer).

Reply:
@jsf9k Made those changes in https://github.com/cisagov/domain-manager-ui/pull/126/commits/98558103e782fa0eb37426c3aa175bf66fa0961c